### ISSUE TITLE
Type Error 1009 - in Editor

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-flash#1283]
**TL;DR:** I saw Type Error 1009 right after deploying when I dragged a script out of my backpack, played it by clicking it, then stopping it. @christanbalch also saw this error in the console when she ran into random projects not playing when she clicked green flag.

=== Details ===

Type Error Projects (all involving same scripts):
- https://scratch.mit.edu/projects/147019078/#editor
- https://scratch.mit.edu/projects/147020145/#editor
- https://scratch.mit.edu/projects/147021077/#editor

**Once you trigger the Type Error, changing the language doesn't work correctly, and the Scratch logo in the upper left doesn't take you to Scratch homepage. Neither does My Stuff.**

I think in my project I narrowed down the issue-causing script to this one:
<img width="263" alt="screen shot 2017-02-24 at 12 17 29 pm" src="https://cloud.githubusercontent.com/assets/1320456/23313357/5cd38f4a-fa8b-11e6-9b50-55e66d16cea6.png">

**Comment by jwzimmer** [comment]
I created project 1000014518 on Staging with random & clone blocks and didn't see any Type Errors. @chrisgarrity added those blocks to a working project on Production and also didn't see any Type Errors.

**Comment by Paddle2See** [comment]
I have two cases on production showing that error - both projects do not appear to run

https://scratch.mit.edu/projects/145939612/
https://scratch.mit.edu/projects/138247639/

I looked at the console and it is showing this 
`Error in event handler for (unknown): TypeError: Cannot read property 'msg' of undefined
    at chrome-extension://fjccjnfkdkdmjohojoggodkigkjkkjhl/contentscript.js:41:10
(unknown) 12:27:05 PM | inf | Refresh Object
(unknown) 12:27:06 PM | inf | SWF Initialized Object
VM3018:1 12:27:13 PM | err | TypeError: Error #1009
sentry.io/:1 POST https://sentry.io/api/19106/store/ 429 ()
VM3022:1 12:27:13 PM | err | TypeError: Error #1009
sentry.io/:1 POST https://sentry.io/api/19106/store/ 429 ()
VM3024:1 12:27:14 PM | err | TypeError: Error #1009`

That first error looks serious - but it is also showing up on another project I looked at that is not showing the TypeError so may not be related.  I will note that the other project (https://scratch.mit.edu/projects/103447663/) was much laggier than I recall even though it appeared to be running and was not showing the TypeError.

**Comment by chrisgarrity** [comment]
Using the set pen color to a color block (not value) causes the buttons not to work
